### ISSUE TITLE
Docs very hard to follow vs old docs

### ISSUE THREAD

**Issue by petervijeh** [BranchMetrics/Branch-Integration-Guides#18]
Hi,

I am trying to implement the journey banner.  

I have it working fine using the old deprecated method:

```
import branch from 'branch-sdk';

import { BRANCH_KEY } from 'constants';
import logoMark from 'assets/logomark.svg';

import './styles.scss';

branch.init(BRANCH_KEY);

branch.banner({
  icon: logoMark,
  title: 'lorem',
  description: 'lorem',
  reviewCount: 1100,
  rating: 4.9,
  iframe: false,
  showMobile: true,
  showDesktop: false,
  disableHide: false,
  forgetHide: false,
});

```

However the new documentation here:

https://docs.branch.io/pages/web/integrate/#create-smart-banner

is very tough to follow. 

How do we implement the banner using the new method,  whatever that is?

**Comment by Sarkar** [comment]
Hi @petervijeh, thanks for sending this. This repo is, unfortunately, deprecated.

Are you looking to create Journeys banners? This is definitely the best doc for that: https://docs.branch.io/pages/web/journeys/

The Web SDK used to have a method for creating banners, but that method is deprecated. It used to fall under .banner(), but now the Journeys product takes control of rendering one. You may also find this repo helpful: http://github.com/branchmetrics/web-sdk

If you still need help, please feel free to reach out to integrations@branch.io -- they'll take care of implementation issues if you run into any.
